p_sets/3/find.rb

```diff
@@ -0,0 +1,56 @@
+module Finder
+  def self.linear_search(needle, hay)
+    max_index   = hay.length - 1
+    result      = false
+    max_index.times do |i|
+      result = true if hay[i] == needle
+    end
+    puts result.to_s
+  end
```

Comment:
I think there's a bug here. Hint: I think `linear_search(1, [1])` will output false.

Two other comments (both ignorable): `puts` will convert everything to string anyway, so you don't need `to_s`. Also when `hay[i] == needle` you might consider `break`ing as well as setting `result = true`. That just saves a bit of time as you don't carry on iterating through the rest of the array when you've already found the needle. E.g. if the input was `linear_search(1, [1,0,0,0,0,0,0,0,0,0,0,0,...])`.